railties/lib/rails/templates/rails/welcome/index.html.erb

```diff
@@ -70,12 +70,31 @@
       position: absolute;
       right: 0;
     }
+
+    header {
+      top: 1rem;
+      left: 0;
+      position: absolute;
+      right: 0;
+    }
+
+    header a {
+      margin: 0.5rem;
+    }
   </style>
 </head>
 
 <body>
+  <header>
```

Comment:
Should we use `nav` here as well?

Reply:
It would conflict with the CSS for `nav` that's used to display the Rails logo.